apps/example/src/Home.tsx

```diff
@@ -27,6 +27,10 @@ export const examples = [
     screen: "Cube",
     title: "🧊 Cube",
   },
+  // {
```

Comment:
Did you mean to leave this commented out?

Reply:
yes I just use this for debugging. Maybe we could enable it and show how the CanvasAPI can be used to run Skia on top of it.